index.html

```diff
@@ -0,0 +1,290 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8" />
+  <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+  <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+  <link rel="stylesheet" href="./css/main.css" />
+  <link rel="icon" href="./favicon.ico" />
+  <title>Aziz Mejri - Portfolio</title>
+</head>
+
+<body>
+                                                  <header>
+                                                    <h1 id="logo">Aziz Mejri</h1>
+                                                    <button type="button" class="menu-bars" id="menu_btn">
+                                                      <i class="fas fa-bars"></i>
+                                                    </button>
+                                                    <nav id="desktop-nav">
+                                                      <ul>
+                                                        <li><a class="nav-link" href="#projects">Portoflio</a></li>
+                                                        <li><a class="nav-link" href="#about-me">About</a></li>
+                                                        <li>
+                                                          <a class="nav-link" href="#contact"
+                                                            >Contact <i class="far fa-envelope"></i
+                                                          ></a>
+                                                        </li>
+                                                      </ul>
+                                                    </nav>
+                                                    <nav id="mobile-nav">
+      <span>x</span>
+      <ul>
+        <li><a class="nav-link" href="#projects">Portfolio</a></li>
+        <li><a class="nav-link" href="#about-me">About</a></li>
+        <li>
+          <a class="nav-link" href="#contact">Contact</a>
+        </li>
+      </ul>
+    </nav>
```

Comment:
Congratulations on great work so far on your design 🚀 , But it does not match with Figma design 💯 .so you should  match your design with the design from[ Figma template](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=1%3A1471). you learning to become professional developer, you should have this ability to follow project guidelines because in future you will need to implement Figma mockups from companies into real projects.

- [ ] Your design :
   
     ![image](https://user-images.githubusercontent.com/90524466/169371402-f10c2d53-b320-45d1-99a2-15efba043212.png)


- [ ] Figma design :


      
![image](https://user-images.githubusercontent.com/90524466/169371816-a5cb3885-debb-4350-91ed-4b95617e8de0.png)